.golangci.yml

```diff
@@ -1,6 +1,10 @@
 run:
   timeout: 5m
-  go: 1.20
```

Comment:
Will this use the go version installed if not defined?

Reply:
https://golangci-lint.run/usage/configuration/

> # Define the Go version limit.
>  # Mainly related to generics support since go1.18.
>  # Default: use Go version from the go.mod file, fallback on the env var `GOVERSION`, fallback on 1.18
>  go: '1.19'

Reply:
i removed it so it uses the one on the repo and not having to update in more places